_docs/0.2.0/usage.md

```diff
@@ -89,9 +89,40 @@ Now how to use it all together:
 
 #### Info File Comparison
 
+Atum's Info file comparison. Ran as part of the E2E Runner but it can be run as a spark job or a plain old jar file.
+
+```shell
+spark-submit info-file-comparison.jar \
+    --ref-path /path/to/reference/data/_INFO/file \
+    --new-path /path/to/new/data/_INFO/file \
+    --out-path /path/to/results
+```
+
+For _INFO file placed in Hadoop use this format of path `hdfs://path/to/_INFO/file`.
+
+##### Properties
+
+Default properties in use are:
+
+```json
+info-file-comparison {
+  atum-models {
+    stdVersionKey = "std_enceladus_version"
+    confVersionKey = "conform_enceladus_version"
+    stdNameKey = "Standartization"
+    confNameKey = "Conformance"
+    stdAppIdKey = "std_application_id"
+    confAppIdKey = "conform_application_id"
```

Comment:
I need to look into this. Seems InfoFileComparison might have more then Atum dependency

Reply:
Please add a Note that this will be changed in #63. So we can merge and we will later rewrite the properties here.